src/zimscraperlib/zim/filesystem.py

```diff
@@ -134,7 +134,10 @@ def make_zim_file(
     redirects: Sequence[tuple[str, str, str]] = None,  # noqa: RUF013  # pyright: ignore
     redirects_file: pathlib.Path = None,  # noqa: RUF013  # pyright: ignore
     rewrite_links: bool = True,  # noqa: FBT001, FBT002, ARG001
-    workaround_nocancel: bool = True,  # noqa: FBT001, FBT002
+    workaround_nocancel: Optional[bool] = True,  # noqa: FBT002
```

Comment:
I don't think we want to expose this. What's the use case? When would we ever want to build a broken ZIM in this mode?